src/features/Conversation/Error/OAuthForm.tsx

```diff
@@ -57,7 +57,7 @@ const OAuthForm = memo<{ id: string }>(({ id }) => {
             block
             icon={<Icon icon={ScanFace} />}
             loading={status === 'loading'}
-            onClick={() => signIn('auth0')}
+            onClick={() => signIn()}
```

Comment:
why we don't need to add `auth0` params?

Reply:
> which is better ?

I would like the `SSO_PROVIDERS=auth0,azure_ad` this one.


> why we don't need to add auth0 params?

I am not a specialist in frontend. I don't know how to let this tsx module read the app config. 
And without this, the user can go to the SSO providers choosing [page](https://lobe.xllb.cc/api/auth/signin?callbackUrl=https%3A%2F%2Flobe.xllb.cc%2Fsettings%2Fcommon). I think this page was provided by NextAuth.js in default.
<img width="432" alt="Screenshot 2024-03-05 at 10 33 08" src="https://github.com/lobehub/lobe-chat/assets/13883964/693587b6-6226-4669-8d6d-a4864b8e8612">



I think the better solution is when the user just set one SSO provider, then this field will be set the same as SSO_PROVIDERS.

If the user has multiple provides, there should be emply to let the user choose the SSO provider.

Reply:
how can I try with this feature?

Reply:
This is my instance https://lobe.xllb.cc/chat

Reply:
@arvinxx If you need, I can make a quick showcase via Zoom, Google meet, or Tencent Meeting.

Reply:
> If you need, I can make a quick showcase via Zoom, Google meet, or Tencent Meeting.

Not need, code can tell everything.

> I think the better solution is when the user just set one SSO provider, then this field will be set the same as SSO_PROVIDERS.

yes I think it's right, but we need a some method to achieve this. Let me think about it.

Reply:
I am working on the document, but I want to split different providers on different pages, is that ok?

And how to insert image? I am not see any image in the repo.